To crop

ErwanJezequel opened the pull request.

Commit:
TOCrop
Added TOCrop iOS

Commit:
TOCrop
Modified README

Commit:
TOCrop
README

dominiquesavoie commented on TOCrop-iOS/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,34 @@
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("erwan.jezequel")]
```
Copyright © 2016 Apcurium 

(note: je vais avoir a fixer can dans les deux autres bindings.)

ErwanJezequel replied:
Modifié

Costo commented on README.md:
```diff
@@ -5,7 +5,22 @@
 Documentation for [Android](https://developers.braintreepayments.com/start/hello-client/android/v2)
 
+## TOCropViewController iOS
+TOCropViewController [GitHub](https://github.com/TimOliver/TOCropViewController)
```
Mettre la description du projet : "A view controller that allows users to crop UIImage objects."

ErwanJezequel replied:
Ajouté

Costo commented:
@ErwanJezequel Essaye de donner plus d'information dans le titre de ta PR et tes messages de commit.

dominiquesavoie commented:
Ajouter une classe partial TOCropViewController avec une methode ShowCropViewAsync

qui obtiens le root viewcontroller en parametre et qui retourne la task du delegate (donc setter le delegate dans la librairie de binding au lieux d'avoir reecrire une partie du code a chaque fois),

ErwanJezequel commented:
ShowCropViewAsync ajoutée

Commit:
Code React
Added a partial class to encapsulate Delegate mechanism
Modified Readme
Added Namespace in ApiDefinition and StructsAndEnum

dominiquesavoie commented on TOCrop-iOS/TOCropViewController.cs:
```diff
@@ -0,0 +1,26 @@
+    public partial class TOCropViewController
+    {
+        /// <summary>
+        /// Shows the crop view async.
```
Shows the crop view and wait for the user's input.

ErwanJezequel replied:
Ajouté

dominiquesavoie commented on TOCrop-iOS/TOCropViewController.cs:
```diff
@@ -0,0 +1,26 @@
+        /// <summary>
+        /// Shows the crop view async.
+        /// </summary>
+        /// <returns>The crop view async.</returns>
```
Returns a byte array of the cropped photo or throws an OperationCancelledException if the user cancelled.

ErwanJezequel replied:
Ajouté

dominiquesavoie commented:
Deux derniers petit commentaires et ensuite tu peux :shipit:

Commit:
TOCrop
Modified code sample to take code react into account

Commit:
TOCrop
Added comments
